spec/requests/api/v1/admin/domain_blocks_spec.rb

```diff
@@ -168,6 +168,16 @@
         expect(response).to have_http_status(422)
       end
     end
+
+    context 'when severity is invalid' do
+      let(:params) { { domain: 'bar.com', severity: :bar } }
+
+      it 'returns http unprocessable entity' do
+        subject
+
+        expect(response).to have_http_status(422)
```

Comment:
Could we also add an assertion about the error json returned in the response?

Reply:
Yeah good idea. I will add one

Reply:
Check. I've added an assertion about the returned JSON.